### DISCUSSION TITLE
'Output Power' auf Null setzen im 'Battery first'-Modus bei voller Batterie

### DISCUSSION THREAD

**Discussion by MartinGoX** [mtrossbach/noah-mqtt#12] [General]
Hallo Markus.
Am Wochenende habe ich meinen NOAH2000 in Betrieb genommen und bin Dank Google auf deine HA-Integration gestoßen. Klasse Arbeit, ich bin begeistert - alle NOAH-Daten sind nun im HA verfügbar.
Etwas hat mich jedoch verwundert am NOAH: ist die Batterie im 'Battery first'-Modus voll geladen und ist weiterhin Solarstrom vorhanden, dann wird dieser ins Haus eingespeist, obwohl ich die System-Standardausgangsleistung auf Null gesetzt habe. Kann ich dem NOAH mittels deiner Intergration dies abgewöhnen - also, wenn Batterie voll, dann nichts weiter einspeisen, erst wenn ein Betriebsmodus 'Verbraucher zuerst' aktiv ist?
Viele Grüße
Martin

**Comment by robinjo86** [comment]
Moin Martin, bin zwar nicht der Entwickler :) aber ich bezweifle das dies umgesetzt werden kann. Das ist ein Feature vom Noah, der lässt seit der vorletzten Firmware die komplette Modulleistung ungebremst durch, sobald die Batterie voll ist, auch LF auf <800 bewirkt da nichts.
Die einzige Möglichkeit das zu handhaben ist die AC Verbindung zu trennen oder die Wirkleistung des WR zu reduzieren. Gerne hier mal vorbeischauen https://www.photovoltaikforum.com/thread/224226-growatt-noah-2000-plug-play-speicher-f%C3%BCr-balkonkraftwerk/?pageNo=244

**Reply by MartinGoX** [reply]
Moin Robin, besten Dank für deine Erläuterung. Ich hatte mir das schon fast gedacht...da hätte Growatt ja auch was zum Einstellen spendieren können...
Den Thread aus dem PV-Forum hatte ich auch schon mal gefunden, nur noch nicht alle 244 Seiten durchgelesen :-)
Viele Grüße, Martin
